src/main/java/seedu/address/ui/UserProfilePanel.java

```diff
@@ -0,0 +1,80 @@
+package seedu.address.ui;
+
+import java.util.logging.Logger;
+
+import com.google.common.eventbus.Subscribe;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.FlowPane;
+import javafx.scene.layout.Region;
+
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.commons.events.model.UserProfileChangedEvent;
+import seedu.address.commons.events.ui.ProfilePictureChangedEvent;
+import seedu.address.model.ReadOnlyAddressBook;
+import seedu.address.model.user.UserProfile;
+
+//@@author {tohcheryl}
+/**
+ * The User Profile panel of the App.
+ */
+public class UserProfilePanel extends UiPart<Region> {
+
+    private static final String FXML = "UserProfilePanel.fxml";
+
+    private static final Logger logger = LogsCenter.getLogger(UserProfilePanel.class);
+
+    private static final String PROFILE_PICTURE_PATH = "file:src/main/resources/images/profilepic.png";
```

Comment:
This is where the problem lies.

When you create the JAR, the whole `src/` package is made into one JAR file, so you cant try paths like `src/main/resources...`.
If you notice, the icon for the app stored in `resources/` is retrieved as `MainApp.class.getResource("address_book_32.png")`, 

So if you need to get anything from `/resource/`, use that method^